tests/integration/tier1/checks-after-conversion/test_rhel_kernel.py

```diff
@@ -1,3 +1,7 @@
+import pytest
+
+
+@pytest.mark.rhel_kernel
 def test_one_kernel_scenario(shell):
```

Comment:
I just noticed that the function name is my old copy paste error :see_no_evil: .Do not take this as a blocker for this PR. It can be changed elsewhere

Reply:
Good catch @kokesak, can you make a ticket for that and assign me to that?

Reply:
@danmyway  here is the ticket https://issues.redhat.com/browse/RHELC-918